packages/types/lib/providers/provider.ts

```diff
@@ -54,6 +54,10 @@ export interface BaseProvider {
             base_url_override?: string;
             headers?: Record<string, string>;
         };
+        authentication_header?: {
+            key: string;
+            value: string;
+        };
```

Comment:
What if you need more than 1 header?
Also the format is different than `headers` which seems to accommodate more than one header, might be a good idea to follow the same pattern